CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ endif()
 
 option(CODE_COVERAGE "Enable code coverage" OFF)
 option(DISABLE_CRASH_LOG "Disable installing easylogging crash handler" OFF)
+option(ENABLE_EXECINFO "Enable execinfo support" OFF)
+
+if(ENABLE_EXECINFO)
+	SET(CMAKE_LINKER_FLAGS "${CMAKE_LINKER_FLAGS} -lexecinfo")
+endif()
```

Comment:
Instead of doing this, please change the logic below that adds execinfo to CORE_LIBRARIES for FreeBSD and NetBSD

Reply:
Since after relooking at this, the issue is mostly the easylogging library from the other vender, so closing.